.changeset/strange-trains-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/remix': patch
+---
+
+Support using env variables in `getAuth` and fix setting resources (`user`/`sessions`/`organization`) into `request`.
```

Comment:
cc: @LekoArts Could you handle adding these changes in clerk-docs?

Reply:
What do you mean by "support using env variables in `getAuth`"?

Reply:
Ignore this. I will replace it with:
> Fix adding `user`/`sessions`/`organization`  resources into `request`

The `support using env variables in getAuth` part was already working. 
PS With that phrase, i meant that i passed the env variables (eg CLERK_SECRET_KEY) as options in the `authenticateRequest` called from `getAuth`.